Mathlib/Algebra/Order/Ring/Int.lean

```diff
@@ -67,6 +67,12 @@ lemma cast_mul_eq_zsmul_cast {α : Type*} [AddCommGroupWithOne α] :
     simp only [sub_mul, one_mul, cast_sub, ih, sub_zsmul, one_zsmul, ← sub_eq_add_neg, forall_const]
 #align int.cast_mul_eq_zsmul_cast Int.cast_mul_eq_zsmul_cast
 
+lemma pos_iff_two_le_of_even {m : ℤ} (even : Even m) : 0 < m ↔ 2 ≤ m :=
+  add_one_le_iff.symm.trans <| Ne.le_iff_lt <| by rintro rfl; exact not_even_one even
```

Comment:
I am adding a very very similar result in #13683. Actually it's more general, so I think you will be able to drop your lemma entirely

Reply:
You mean [this lemma](https://github.com/leanprover-community/mathlib4/blob/a493200aadc17ca309db1e5e14cecd7a8841d725/Mathlib/Data/Nat/Defs.lean#L1133-L1135)? It's about Nat rather than Int though. I can add the Int version of the lemma to this PR.